package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "jslings",
+  "version": "0.0.1",
+  "description": "The easiest way to get started with JavaScript. 🐢",
+  "main": "index.js",
+  "repository": "git@github.com:XifeiNi/jslings.git",
+  "author": "Anthony Morris <anthony@amorrissound.com>",
+  "contributors": [],
+  "license": "MIT",
+  "devDependencies": {
```

Comment:
Can we add Typescript? And setup the basic scripts, like start in dev mode and start?

Reply:
I installed TypeScript but left the scripts. If you choose [oclif](https://oclif.io) it might be different vs. doing something else? Hope that's okay!